### PR TITLE
Dev/1.10.0+1.21

### PR DESCRIPTION
**Pull request by TauCu**
Currently waiting for Architectury API for forge 1.20.6

### CODE REVIEW

**Comment by TauCubed** [comment]
For those too impatient for the 1.20.6 fabric version here is a jar:
[brb-1.10.1+1.20.6.zip](https://github.com/mrshmllow/BetterRecipeBook/files/15217148/brb-1.10.1%2B1.20.6.zip)

It hasn't been fully tested and might not work/break things.

**Comment by sternschnaube** [comment]
Thank you @TauCubed! 😊

File: gradle.properties [hunk 1]
```diff
@@ -1,21 +1,21 @@
 org.gradle.jvmargs=-Xmx2048M
 
-minecraft_version=1.20.4
+minecraft_version=1.20.6
 enabled_platforms=fabric,forge
 
 archives_base_name=brb
-mod_version=1.10.0+1.20.4
+mod_version=1.10.1+1.20.6
```
Comment (mrshmllow) [inline gradle.properties:7]: Do we need to bump? I have never released 1.10.0 besides "rc"s

Comment (TauCubed) [reply gradle.properties:7]: That's fair enough. I had assumed it was the released version.

**Comment by TauCubed** [comment]
I'm taking it out of a draft state as the common+fabric mod seems to be stable. Haven't tested neoforge much though.

**Comment by mrshmllow** [comment]
Im down to drop forge entirely and just use neoforge...

**Comment by TauCubed** [comment]
You could, the issue is many people still use forge.

**Comment by mrshmllow** [comment]
i dont use *forge, i heard they where compatible though

**Comment by TauCubed** [comment]
Just dropping some compiled 1.21 binaries here until release. For both fabric and neoforge.
These haven't been tested and might break things.

[brb.zip](https://github.com/user-attachments/files/15831735/brb.zip)

**Comment by mrshmllow** [comment]
If I have time this weekend I'll review

**Comment by mrshmllow** [comment]
maybe this weekend.... 🥲

**Comment by TauCubed** [comment]
forgot to make it as ready for review.

**Review by mrshmllow — APPROVED** [review]

**Pull request merged**